### PR TITLE
[12.0][MIG] purchase_request_analytic

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

File: purchase_request_analytic/views/purchase_request_views.xml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="purchase_request_analytic_view_tree" model="ir.ui.view">
+        <field name="name">purchase.request.analytic.tree</field>
+        <field name="model">purchase.request</field>
+        <field name="inherit_id" ref="purchase_request.view_purchase_request_tree"/>
+        <field name="arch" type="xml">
+            <field name="date_start" position="after">
+                <field name="analytic_account_id" groups="purchase.group_analytic_accounting"/>
```
Comment (AaronHForgeFlow) [inline purchase_request_analytic/views/purchase_request_views.xml]: purchase.group_analytic_accounting does not exist :-1:

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]

**Review by AaronHForgeFlow — COMMENTED** [review]
It works as expected. May I ask for an improvement? 

If I create a purchase request and I put an analytic account, then purchase request lines are not defaulted to that analytic account. That would be nice. Apart of this everything works fine.

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Code review + Functional test

File: purchase_request_analytic/models/purchase_request.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    _inherit = 'purchase.request'
+
+    analytic_account_id2 = fields.Many2one(
+        comodel_name='account.analytic.account',
+        help='Use to store the value of analytic_account if there is no lines')
+    analytic_account_id = fields.Many2one(
+        compute='_compute_analytic_account_id',
+        inverse='_inverse_analytic_account_id',
+        comodel_name='account.analytic.account',
+        string='Contract / Analytic', readonly=True,
```
Comment (CasVissers-360ERP) [inline purchase_request_analytic/models/purchase_request.py]: Why not use string='Analytic Account' like on other places in Odoo?

Comment (MiquelRForgeFlow) [reply purchase_request_analytic/models/purchase_request.py]: done!

**Review by CasVissers-360ERP — APPROVED** [review]
LGTM 👍

**Pull request merged**